src/core/issues-notes.js

```diff
@@ -23,7 +23,18 @@ export function run(conf, doc, cb) {
         "<div><h2>" + conf.l10n.issue_summary + "</h2><ul></ul></div>"
       ),
       $issueList = $issueSummary.find("ul");
+    Array.from($ins)
+      .filter(({ dataset: { number } }) => number && !ghIssues[number] && conf.githubAPI)
+      .forEach(elem => {
+        const { dataset: { number } } = elem;
+        const msg = `Github issue ${number} was closed on GitHub, so removing from spec`;
+        pub("warn", msg);
+        elem.remove();
+      });
     $ins.each(function(i, inno) {
+      if (!document.contains(inno.parentNode)) {
```

Comment:
Oops, what I meant was `if (!inn.parentNode)` as any removed elements does not have a parent node.

Reply:
Oh sorry, even I intended on writing the same but maybe I might have accidentally written that